### PR TITLE
chore(nifi): update versions for 24.11

### PR DESCRIPTION
**Pull request by razvan**
# Description

part of https://github.com/stackabletech/docker-images/issues/836

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes

```[tasklist]
- [ ] Changes are OpenShift compatible
- [ ] All added packages (via microdnf or otherwise) have a comment on why they are added
- [ ] Things not downloaded from Red Hat repositories should be mirrored in the Stackable repository and downloaded from there
- [ ] All packages should have (if available) signatures/hashes verified
- [ ] Add an entry to the CHANGELOG.md file
- [ ] Integration tests ran successfully
```

<details>
<summary>TIP: Running integration tests with a new product image</summary>

The image can be built and uploaded to the kind cluster with the following commands:

```shell
bake --product <product> --image-version <stackable-image-version>
kind load docker-image <image-tagged-with-the-major-version> --name=<name-of-your-test-cluster>
```

See the output of `bake` to retrieve the image tag for `<image-tagged-with-the-major-version>`.
</details>

### CODE REVIEW

File: nifi/versions.py
Comment (NickLarsenNZ) [inline nifi/versions.py:1]: The comment in this file might need updating"

```diff
- "java-devel": "11",  # There is an error when trying to use java-devel 21 (for nifi 1.26.0):
+ "java-devel": "11",  # There is an error when trying to use java-devel 21 (for nifi 1.27.0):
```

Or perhaps

```diff
- "java-devel": "11",  # There is an error when trying to use java-devel 21 (for nifi 1.26.0):
+ "java-devel": "11",  # There is an error when trying to use java-devel 21 (since nifi 1.26.0):
```

**Review by NickLarsenNZ — APPROVED** [review]
LGTM, just a note about a comment that seems a bit off.

File: nifi/versions.py
Comment (razvan) [reply nifi/versions.py:1]: Changed: https://github.com/stackabletech/docker-images/pull/868/commits/70f0f13e4626328c69ade45b4ce40ffc71126b46

**Review by NickLarsenNZ — APPROVED** [review]
LGTM

**Pull request merged**